code/datums/mutations.dm

```diff
@@ -129,7 +129,8 @@
 	var/status = CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
 	owner.status_flags &= ~status
 	owner.update_body_parts()
-
+	owner_dna = owner.dna // fukken callbacks
```

Comment:
you quite literally could not have gotten "undefined var owner_dna" when you didn't write that before.